data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg

```
@@ -160,7 +160,8 @@
 
         [message]
             speaker=Hamel
-            message= _ "As you can see, Tallin, there are muckle heaps of them, all ready to be borne into battle. What’s more, some of the dwarves who are expert with the sword, mace and bow, as well as some of yer own men who have grown skilled by experience, can teach others how to wield them."
+            # po: As you can see, Tallin there are a lot of them ready to be born into battle. Some of us dwarves are experts in the sword, mace, and bow and can aid your veterans in teaching others.
+            message= _ "As ye ken see, Tallin, there be muckle heaps o' them, all ready to be borne inta battle. And, some o’ us dwarves be experts in the sword, mace, and bow and ken aid yer veterans in teaching others."
```

Just realized this apostrophe needs to be the unicode one.

```suggestion
            message= _ "As ye ken see, Tallin, there be muckle heaps o’ them, all ready to be borne inta battle. And, some o’ us dwarves be experts in the sword, mace, and bow and ken aid yer veterans in teaching others."
```